command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 08:11:40PM -0700, Doug Davis wrote:
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> Change:
> ```
> A conformant [runtime][] MUST provide an executable...
> ```
> to something like:
> ```
> A compliant implementation of this "command line interface"
> specification MUST provide an executable...
> ```
> To make it clear that we're talking about being compliant with the
> CLI spec, not the RUNTIME spec.

Once I get a URL back from the Trademark Board / Compliance Working
Group, I intend to put in some better compliance-testing language [1].
But I think *all* MUSTs in this runtime-spec are granular like that
[2].  For example, ccon can be compliant with config.md,
config-linux.md, and command-line-interface.md without being compliant
with config-solaris.md.  So I don't see a need (beyond the better
compliance-testing language [1]) to underline the granularity for this
particular portion of the spec.  If that granularity needs additional
underlining, I suggest we do it around the “the protocols it
implements” wording in [3].

However, granular compliance with some portions of runtime-spec in the
absence of compliance with an OCI-specified API will not be
OCI-certifiable [2].  Assuming something like this command line API
lands, runtimes MUST implement it (although they are free to *also*
implement other interfaces [1,4]) if they want to be OCI certified as
compliant.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72713742
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72693334
[3]: https://github.com/opencontainers/runtime-spec/tree/v1.0.0-rc1#open-container-runtime-specification
[4]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72703237